molecule/scenario_resources/Dockerfile.j2

```diff
@@ -61,6 +61,7 @@ dnf --assumeyes install \
   bash \
   jemalloc \
   glibc-langpack-en \
+  libxcrypt-compat \
```

Comment:
I'd really like a comment on why we're installing this.

And basically we shouldn't be installing very much in the dockerfile anyway. It should be in the installer, unless it's something Ansible needs to connect in the 1st place (python3), or there's some other exceptional reason.

Reply:
looks this was local issue, removed